source/includes/kubernetes_extension/_k8_persistentvolumes.md

```diff
@@ -246,6 +246,89 @@ Return value:
 | `taskId` <br/>_string_     | The id corresponding to the create persistent volume task.       |
 | `taskStatus` <br/>_string_ | The status of the operation.                                     |
 
+<!-------------------- REPLACE PERSISTENT VOLUME -------------------->
+
+#### Replace a persistent volume
+
+```shell
+curl -X PUT \
+  -H "MC-Api-Key: your_api_key" \
+   "https://cloudmc_endpoint/v1/services/a_service/an_environment/persistentvolumes/my-persistent-volume?cluster_id=a_cluster_id"
+  Content-Type: application/json
+  {
+    "apiVersion": "v1",
+    "kind": "PersistentVolume",
+    "metadata": {
+        "annotations": {
+            "key": "value"
+        },
+        "name": "persistentvolumes-name"
+    },
+    "spec": {
+        "accessModes": [
+            "ReadWriteMany"
+        ],
+        "capacity": {
+            "storage": "2Gi"
+        },
+        "claimRef": {
+            "apiVersion": "v1",
+            "kind": "PersistentVolumeClaim",
+            "name": "pv-claim-name",
+            "namespace": "default",
+            "resourceVersion": "259087",
+            "uid": "11957feb-1acc-4c7e-911e-f77d5d7be5ac"
+        },
+        "hostPath": {
+            "path": "/etc/path"
+        },
+        "persistentVolumeReclaimPolicy": "Retain",
+        "storageClassName": "standard",
+        "volumeMode": "Filesystem"
+    }
+}
+```
+
+> The above command returns a JSON structured like this:
+
+```json
+{
+  "taskId": "1542bd45-4732-419b-87b6-4ea6ec695c2b",
+  "taskStatus": "PENDING"
+}
+```
+
+<code>PUT /services/<a href="#administration-service-connections">:service_code</a>/<a href="#administration-environments">:environment_name</a>/persistentvolumes/:id?cluster_id=a_cluster_id</code>
+
+Replace a persistent volume in a given [environment](#administration-environments).
+
+Required Attributes                 | &nbsp;
```

Comment:
I think we use `Attributes` as it implicitly means required attributes. Can we check with other sections of the API docs to make sure that it's consistent across the doc?

Reply:
Its Required attributes in other sections

Reply:
Thanks for checking that.